src/base/preferences.cpp

```diff
@@ -1054,6 +1056,115 @@ void Preferences::setExecutionLogEnabled(bool b)
     setValue("Preferences/ExecutionLog/enabled", b);
 }
 
+bool Preferences::executionLogNormalType() const
```

Comment:
Good idea but QVariant doesn't have a constructor for that. I am thinking of a QBitArray.